src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier supplier = new ColorSupplier();
+        ball.setColor(String.valueOf(supplier.getRandomColor()));
+        Random number = new Random();
```

Comment:
```suggestion
    private Random random;
    private ColorSupplier colorSupplier;
    
    public String getRandomBall() {
        Ball ball = new Ball();
        ball.setColor(String.valueOf(supplier.getRandomColor()));
```
Better make Random and ColorSupplier class fields and initialize it using constructor.